### DISCUSSION TITLE
fluXpipe

### DISCUSSION THREAD

**Discussion by amotl** [crate/crate-clients-tools#308] [Ideas]
## About
TLDR; Flux for InfluxDB 3.0. Stand-alone, Serverless Flux API/Pipeline for querying, analyzing and interacting with remote data. 

## Details
Coming from a recent discussion about InfluxDB with @hlcianfagna and @wierdvanderhaar, it will be nice to see a few code examples using [fluXpipe](https://github.com/metrico/fluXpipe) to move data around between MQTT, InfluxDB, PostgreSQL, and CrateDB.

## Resources
- https://github.com/metrico/fluXpipe
- https://fluxpipe.fly.dev/

## References
- https://github.com/influxdata/flux/issues/4389
- https://github.com/influxdata/flux/issues/5410
- https://github.com/influxdata/flux/pull/5370
- https://github.com/influxdata/flux/pull/5431
- https://github.com/metrico/fluXpipe/tree/main/scripts
